docs/topics/security.rst

```diff
@@ -25,7 +25,9 @@ ASGI middlewares it contains, ``OriginValidator`` and
 ``OriginValidator`` lets you restrict the valid options for the ``Origin``
 header that is sent with every WebSocket to say where it comes from. Just wrap
 it around your WebSocket application code like this, and pass it a list of
-valid domains as the second argument::
+valid domains as the second argument. You can pass only domain (for example,
+``.allowed-domain.com``) and full origin, than must be scheme://domain[:port]
```

Comment:
"and full origin, than must be" -> "or a full origin, in the format" (and make the format code-style formatting)